packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -181,6 +181,14 @@ class DriveCommand extends RunCommandBase {
     ResidentRunner residentRunner;
     final BuildInfo buildInfo = getBuildInfo();
     final bool isWebPlatform = await device.targetPlatform == TargetPlatform.web_javascript;
+    final File applicationBinary = stringArg('use-application-binary') == null
```

Comment:
Not for this PR, but in general, how will we handle this pattern once switching to null-safety?

Reply:
Yeah, in the places where we use null as a sentinel today we'd still use after null safety. It will just be harder to forget to check for it.